docs/collaboration/governance.md

```diff
@@ -280,35 +280,35 @@ OpenStand:
 
 ### B. Standard decision making process overview
 
-The foundation of BIDS decision making is listening to all members of
-the BIDS Community and striving to achieve consensus on each level of
-the BIDS standard process.
+The foundation of BIDS decision making is listening to all members of the BIDS Community
+and striving to achieve consensus on each level of the BIDS standard process.
 
 The criteria for forming a new BEP Working Group:
 
 -   Statement of intent with defined scope, deliverables, and use cases
 -   Statement of intent accepted by the BIDS Steering Group
 
-The criteria of the BIDS Steering Group evaluation for elevating a Draft
-BEP to a Proposed BEP:
+The criteria of the BIDS Steering Group evaluation
+for elevating a Draft BEP to a Proposed BEP:
 
 -   Integrates into the current edition of the standard and is consistent with the BIDS Mission Statement
 -   Clearly defines a filename template and field names
 -   At least 10 business days of community feedback publicly posted across the BIDS channels
 -   Consensus reached among the Working Group
 -   Use cases and examples clearly illustrated
 
-Upon a successful Draft BEP review, the BEP will be converted from a
-Google document to a pull request for the
-[BIDS standard][specification_gh].
-This will enable further community feedback on the Proposed BEP. Tools
-may begin integrating the Proposed BEP specification.
+Upon a successful Draft BEP review,
+the BEP will be converted from a Google document
+to a pull request for the [BIDS standard][specification_gh].
+This will enable further community feedback on the Proposed BEP.
```

Comment:
Is this the place to put "an announcement should be made to bids-discussion" ... as well as above?